features/show_dishes.feature

```diff
@@ -7,6 +7,9 @@ Feature: user can see dishes and their info
     Given the following dishes exists
       | name           | description           | price  |  pic_url                | restaurant | category |
       | Sushi rolls    | Tasty Japanese food   | 10     |  https://goo.gl/fH7P5F  | Ai         | Main     |
+    And the following restaurants exist
+      | name    | address          |
+      | My Thai | Stockholm        |
```

Comment:
Will you connect dishes to a restaurant in a different story? At this point this scenario feels a bit weird

Reply:
This story is about clicking a link on main page and getting to the restaurant page, we were just trying to stick to the story and do nothing outside of it

Reply:
Good thinking I all comment responses. Good job